src/resumes/left-right.vue

```diff
@@ -13,7 +13,7 @@
   </div>
   <div class="left half">
     <div class="experience">
-      <h3>Experience</h3>
+      <h3>{{ terms[person.lang].headings.experience }}</h3>
```

Comment:
Maybe make a computed property so this stuff becomes a bit cleaner.
```
 <h3>{{ localizedPerson.headings.experience }}</h3>
```
```js
computed:{
 localizedPerson(){
   return this.terms[person.lang];
 }
}
```

Maybe it can be renamed to something shorter.

Reply:
This has to be added to the specific vue template, am I right?
If yes, is it possible to outsource this small function? Because it has to be added to every template. I think the templates should contain as little JS code as possible to reduce redundancy. Not 100% familiar with vue.

Reply:
You can make a mixin and declare the data there as well and then only import the mixin in each template. This will probably be easier for other people when adding templates because all of them would share the same logic/data

Reply:
To have minimal code in the template itself, the whole component data could be outsourced.
Would something like this work (scopes etc)? It would reduce redundancy a lot.
```javascript
// my-template.vue
...
import { getVueComponent } from './resume.js';

export default getVueComponent('my-template');
...
// resume.js
...
import { PERSON } from '../person';
import { terms } from '../terms';

function getVueComponent (name) {
  return {
    name: name,
     data () {
        return {
          person: PERSON,
          terms: terms
        };
      }
    },
    computed:{ 
      localizedPerson(){
        return this.terms[person.lang];
      }
    }
  };
}

export getVueComponent;
```

Reply:
I think your option of having a separate function which would be imported in each template is good. 👍

Reply:
@salomonelli would this be ok or are there things necessary to proceed directly in the template?

Reply:
Should be fine!